### ISSUE TITLE
Use downloaded model file

### ISSUE THREAD

**Issue by Hongtao-Xu** [NUS-HPC-AI-Lab/VideoSys#185]
Great work!!
How can i use the locally downloaded model file in Open Sora?
Can you give me a example, please?

**Comment by Hongtao-Xu** [comment]
I tried using the following code, but the result video is a snowflake：
```python
from videosys import OpenSoraConfig, VideoSysEngine
import time

def run_base():
    config = OpenSoraConfig(world_size=1,
                            model_path="checkpoints/OpenSora_checkpoint/OpenSora-STDiT-v3",
                            vae="checkpoints/OpenSora_checkpoint/OpenSora-VAE-v1.2",
                            text_encoder="checkpoints/OpenSora_checkpoint/t5-v1_1-xxl",
                            )
    engine = VideoSysEngine(config)

    prompt = "Sunset over the sea."
    video = engine.generate(prompt).video[0]
    engine.save_video(video, f"./outputs/opensora_{prompt}_base.mp4")


def run_pab():
    config = OpenSoraConfig(world_size=1, enable_pab=True)
    engine = VideoSysEngine(config)

    prompt = "Sunset over the sea."
    video = engine.generate(prompt).video[0]
    engine.save_video(video, f"./outputs/opensora_{prompt}_pab.mp4")


if __name__ == "__main__":
    t1 = time.time()
    run_base()
    t2 = time.time()
    run_pab()
    t3 = time.time()
    print("Base={:.3f}".format(t2-t1))
    print("PAB={:.3f}".format(t3-t2))
```
and I also  change this code:
```python
class VideoAutoencoderPipeline(PreTrainedModel):
    config_class = VideoAutoencoderPipelineConfig

    def __init__(self, config: VideoAutoencoderPipelineConfig):
        super().__init__(config=config)
        self.spatial_vae = VideoAutoencoderKL(
            from_pretrained="checkpoints/OpenSora_checkpoint/pixart_sigma_sdxlvae_T5_diffusers", # my local dir
            local_files_only=False,
            micro_batch_size=4,
            subfolder="vae",
        )
        self.temporal_vae = VAE_Temporal_SD()
        self.cal_loss = config.cal_loss
        self.micro_frame_size = config.micro_frame_size
        self.micro_z_frame_size = self.temporal_vae.get_latent_size([config.micro_frame_size, None, None])[0]
```

**Comment by oahzxl** [comment]
i suppose "checkpoints/OpenSora_checkpoint/OpenSora-STDiT-v3" this is a wrong ckpt. it should be v1.2 now

**Comment by Hongtao-Xu** [comment]
Thanks for response!!
"checkpoints/OpenSora_checkpoint/OpenSora-STDiT-v3" is from: https://huggingface.co/hpcai-tech/OpenSora-STDiT-v3
Is it possible that my example code miss something? or can you give me a example that use local ckpt?

**Comment by oahzxl** [comment]
you can follow the init code here, just replace model path: https://github.com/NUS-HPC-AI-Lab/VideoSys/blob/51045686109b6982afbde0c49d14e27843db4f2f/videosys/pipelines/open_sora/pipeline_open_sora.py#L154

**Comment by oahzxl** [comment]
fixed now #202
